app/models/award.rb

```diff
@@ -0,0 +1,8 @@
+class Award < ApplicationRecord
+  validates :name, presence: true, uniqueness: true, length: { maximum: 100 }
+  validates :description, presence: true
+  validates :award_type, presence: true
```

Comment:
Use a different column name that doesn't use "type" please.

Reply:
Just made the changes.